### PR TITLE
Dual Helicopter setup wiki

### PR DESCRIPTION
**Pull request by bnsgeyer**
This PR adds a basic setup wiki for dual helicopters.

### CODE REVIEW

File: copter/source/index.rst [hunk 1]
```diff
@@ -193,6 +193,7 @@ Getting more info
    Traditional Helicopters <docs/traditional-helicopters>
    Tricopter <docs/tricopter>
    SingleCopter and CoaxCopter <docs/singlecopter-and-coaxcopter>
+   Dual Helicopters <docs/dual_helicopter>
```
Comment (Hwurzburg) [inline copter/source/index.rst:196]: build error
```suggestion
   Dual Helicopters <docs/dual-helicopter>
```

File: copter/source/docs/dual-helicopter.rst [hunk 2]
```diff
@@ -0,0 +1,206 @@
+.. _dual_helicopter:
```
Comment (Hwurzburg) [inline copter/source/docs/dual-helicopter.rst:1]: build error
```suggestion
.. _dual-helicopter:
```

File: copter/source/docs/dual-helicopter.rst [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+This setup assumes that the forward rotor swashplate is controlled by servo output 1, 2, and 3 and the aft rotor swashplate is controlled by servo output 4, 5 and 6.
+
+Parameter Settings:
+
+- :ref:`H_DUAL_MODE<H_DUAL_MODE>` - Set to 0 for the longitudinal configuration
+- :ref:`H_DCP_SCALER<H_DCP_SCALER>` - Scales the differential collective output to the rotors for a pitch axis input
+- :ref:`H_DCP_YAW<H_DCP_YAW>` - Feedforward input to yaw axis for pitch axis (DCP) inputs
+- :ref:`H_YAW_SCALER<H_YAW_SCALER>` - Scales the differential lateral cyclic based on yaw axis input.  This is a positive number for this setup.
+- :ref:`H_COL_MAX<H_COL_MAX__AP_MotorsHeli_Dual>`- PWM for maximum collective pitch on forward rotor head that corresponds to :ref:`H_COL_ANG_MAX<H_COL_ANG_MAX>`
```
Comment (Hwurzburg) [inline copter/source/docs/dual-helicopter.rst:35]: not defined in Dual...
```suggestion
- :ref:`H_COL_MAX<H_COL_MAX_>`- PWM for maximum collective pitch on forward rotor head that corresponds to :ref:`H_COL_ANG_MAX<H_COL_ANG_MAX>`
```

Comment (Hwurzburg) [reply copter/source/docs/dual-helicopter.rst:35]: similarly below also

File: copter/source/docs/dual-helicopter.rst [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+- :ref:`H_COL_MIN<H_COL_MIN__AP_MotorsHeli_Dual>` - PWM for minimum collective pitch on forward rotor head that corresponds to :ref:`H_COL_ANG_MIN<H_COL_ANG_MIN>`
```
Comment (Hwurzburg) [inline copter/source/docs/dual-helicopter.rst:36]: not defined in Dual
```suggestion
- :ref:`H_COL_MIN<H_COL_MIN>` - PWM for minimum collective pitch on forward rotor head that corresponds to :ref:`H_COL_ANG_MIN<H_COL_ANG_MIN>`
```
similarly below also

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
Just build error review....will read for content later....moving a bit slow with COVID right now

File: copter/source/docs/dual-helicopter.rst (hunk 2 above)
Comment (bnsgeyer) [reply copter/source/docs/dual-helicopter.rst]: fixed. Thanks!

File: copter/source/docs/dual-helicopter.rst (hunk 3 above)
Comment (bnsgeyer) [reply copter/source/docs/dual-helicopter.rst]: fixed. Thanks!

**Comment by Hwurzburg** [comment]
should we point the reader to some setup sections in single blade to help then set collective ranges, etc. for the parameter values? if only to give them some clue

**Comment by Hwurzburg** [comment]
sorry, COVID delay....added a few notes....merging

**Comment by bnsgeyer** [comment]
@Hwurzburg no problem.  I appreciate the help. I will look at this again when I get time to see if I can make additional improvements so it provides a better flow. I’m glad we were able to get this in. There really isn’t anything out there that helps users with dual heli’s.

**Pull request merged**